advanced/am-pattern-3/templates/am/control-plane/instance-2/wso2am-pattern-3-am-control-plane-deployment.yaml

```diff
@@ -32,25 +32,24 @@ spec:
       labels:
         deployment: {{ template "am-pattern-3.resource.prefix" . }}-am-cp
         node: {{ template "am-pattern-3.resource.prefix" . }}-am-cp-2
+        product: apim
     spec:
       initContainers:
-        {{ if .Values.wso2.deployment.dependencies.mysql }}
-        - name: init-mysql-db
+        - name: init-db
           image: busybox:1.32
-          command: ['sh', '-c', 'echo -e "Checking for the availability of MySQL Server deployment"; while ! nc -z "wso2am-mysql-db-service" 3306; do sleep 1; printf "-"; done; echo -e "  >> MySQL Server has started";']
-        - name: init-mysql-connector-download
+          command: ['sh', '-c', 'echo -e "Checking for the availability of DB Server deployment"; while ! nc -z "{{ .Values.wso2.deployment.am.cp.db.hostname }}" {{ .Values.wso2.deployment.am.cp.db.port }}; do sleep 1; printf "-"; done; echo -e "  >> DB Server has started";']
+        - name: init-db-connector-download
           image: busybox:1.32
           command:
             - /bin/sh
             - "-c"
             - |
               set -e
               connector_version=8.0.17
-              wget https://repo1.maven.org/maven2/mysql/mysql-connector-java/${connector_version}/mysql-connector-java-${connector_version}.jar -P /mysql-connector-jar/
```

Comment:
Formatting issue in alignment

Reply:
Fixed in : https://github.com/wso2/kubernetes-apim/pull/560